src/NServiceBus.NHibernate/Subscriptions/NHibernateSubscriptionStorage.cs

```diff
@@ -34,20 +38,16 @@ protected override void Setup(FeatureConfigurationContext context)
             builder.AddMappings<SubscriptionMap>();
             var config = builder.Build();
 
-            Func<string, Task> installAction = _ => Task.FromResult(0);
-
             if (RunInstaller(context))
             {
-                installAction = identity =>
+                context.Settings.Get<Installer.SchemaUpdater>().Execute = identity =>
```

Comment:
Not a big fan of the Execute setter and then overriding it. But I see it is a necessary evil. The other solution would be to create a default empty one in the settings and overwrite it with a new instance of an Updater when necessary. I would personally prefer the later approach